users/models.py

```diff
@@ -8,3 +10,34 @@ class Person(models.Model):
 
     class Meta:
         abstract = True
+
+
+class Club(models.Model):
+    name = models.CharField(_('name'),
+                            max_length=128)
+    contact = models.OneToOneField('MainCoach',
+                                   on_delete=models.CASCADE)
+    street = models.CharField(_('street'),
+                              max_length=128,
+                              blank=True)
+    number = models.PositiveSmallIntegerField(_('number'),
+                                              blank=True)
+    number_extension = models.CharField(_('number extension'),
+                                        max_length=10,
+                                        blank=True)
+    po_box = models.CharField(_('PO Box'),
```

Comment:
denk je dat dit nodig is? Je kan ook vaak gewoon een Postbus 51 als adres toevoegen toch?